app.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+let express = require('express');
+let path = require('path');
+let favicon = require('serve-favicon');
```

Comment:
Ah ok, that makes sense then.

Yeh, let's remove these and just do a hello world in a simple `get('/')`. It will be easier for people to understand and follow along I think?